index.html

```diff
@@ -238,7 +238,7 @@ <h3>Uber Navigation</h3>
             
 
```

Comment:
#6 Kindly remove double or empty line spaces that are unnecessary to make your code more clean.